### ISSUE TITLE
 [WinError 126] The specified module could not be found

### ISSUE THREAD

**Issue by TrinhTuanHung2021** [lyft/l5kit#307]
Hello.
I am using python 3.9. When installing l5kit with the command "pip install l5kit", there is no error. But when I import the l5kit.evaluation module, it says that this module is not available. [WinError 126] The specified module could not be found.
This error only occurs when I run it on Visual Studio Code, but it works well on Google Colab. I try with python 3.8 and 3.7 but can not correct it. How would I fix it? Thanks

**Comment by lucabergamini** [comment]
Sorry for the late reply,

Are you on windows? If that's the case I'm not really sure I can help you with that.
I would check the `PYTHONPATH` to start with

**Comment by TrinhTuanHung2021** [comment]
Thank you for replying to me.
I have solved that problem already. I installed l5kit version 1.2.0 and then updated it to 1.3.1 and it works fine. I also don't know why. Now I have finished running the file in the visualization folder and agent_motion_prediction. But I cannot run the file train.ipynb in the planning folder. It gives the following error 

![image](https://user-images.githubusercontent.com/82990411/128699383-1fcc2c6f-a574-4181-88f0-89cd8f49f9f0.png)
